test/typecheck/main.go

```diff
@@ -105,6 +105,9 @@ func newAnalyzer(platform string) *analyzer {
 		ctx.BuildTags = append(ctx.BuildTags, tagsSplit...)
 	}
 
+	// add selinux tag explicitly
+	ctx.BuildTags = append(ctx.BuildTags, "selinux")
```

Comment:
should we be adding this on all platforms?
selinux seems goofy on ... not linux 🙃

Reply:
the `!linux` kicks in https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/opencontainers/selinux/go-selinux/label/label.go#L1

Reply:
Fair, so we expect any other code to use this tag?

Reply:
dunno, since we have a bunch of folks on Mac, we will know very soon and can adjust then.

Reply:
as long as we're not going to leverage the tag ourselves, this seems fine.

Reply:
sounds good!